src/services/resource.js

```diff
@@ -1759,4 +1761,42 @@ module.exports = class resourceHelper {
 			throw error
 		}
 	}
+
+	/**
+	 * Fetch the consumption deep link for a solution
+	 * @param {string} solutionId - The ID of the solution to fetch the deep link for.
+	 * @param {string} solutionType - The type of the solution (e.g., 'program', 'project').
+	 * @returns {Promise<Object>} - Resolves with the deep link response object.
+	 */
+	static async getDeepLink(solutionId, solutionType) {
+		let result = {
+			deepLinks: '',
```

Comment:
@adithyadinesh0412 if error throw why we need to give the deeplink in result. We can send 500 error without result